openwisp_monitoring/check/apps.py

```diff
@@ -1,8 +1,38 @@
 from django.apps import AppConfig
+from django.conf import settings
+from django.core.cache import cache
 from django.utils.translation import ugettext_lazy as _
+from django_netjsonconfig.signals import checksum_requested, config_download_requested
 
 
 class CheckConfig(AppConfig):
     name = 'openwisp_monitoring.check'
     label = 'check'
     verbose_name = _('Network Monitoring Checks')
+
+    def ready(self):
+        from openwisp_controller.config.models import Device
```

Comment:
since this code is related to a Device, I'd move this to `openwisp_monitoring.device`